AUTHORS

```diff
@@ -109,6 +114,7 @@ Caleb Spare <cespare@gmail.com>
 Calen Pennington <cale@edx.org>
 Cameron Boehmer <cameron.boehmer@gmail.com>
 Carl X. Su <bcbcarl@gmail.com>
+cc272309126 <cc272309126@gmail.com>
```

Comment:
I found this guy via google search as Chen Chao. I'm assuming I should update .mailmap with his correct name